cli/src/config/templates.toml

```diff
@@ -156,12 +140,26 @@ commit_summary_separator = 'label("separator", " | ")'
   time_range.start().ago() ++ label("time", ", lasted ") ++ time_range.duration()'''
 'format_timestamp(timestamp)' = 'timestamp.local().format("%Y-%m-%d %H:%M:%S")'
 
+format_op_log_operation = '''
+  label(if(current_operation, "current_operation"),
+    concat(
+      separate(" ", id.short(), user, format_time_range(time)), "\n",
+      description.first_line(), "\n",
+      if(tags, tags ++ "\n"),
+    ),
+  )
+'''
+format_op_log_snapshot = 'format_op_log_operation'
+# the 'builtin' version is deprecated
+'builtin_op_log_root(op_id)' = 'separate(" ", op_id.short(), label("root", "root()")) ++ "\n"'
```

Comment:
I personally don't think `builtin_*_root()` templates are public interface, so they can be freely removed (with some changelog entries.)

If we need to keep them as deprecated, maybe we can insert inline deprecation message `label("warning", "<Warning: deprecated ..>")`?

Reply:
Well initially I've added the separate builtin_log_root() thinkg specifically so I could override it :upside_down_face: 

I was ok with freely removing them, but then I thought what if I'm not the only person using that one.

I guess just removing with a changelog entry is fine